example/mnist/mnist/custom_evaluator.py

```diff
@@ -0,0 +1,115 @@
+import typing as t
+from types import TracebackType
+from pathlib import Path
+
+import numpy as np
+import torch
+from PIL import Image as PILImage
+from loguru import logger
+from torchvision import transforms
+
+from starwhale import (
+    step,
+    Image,
+    Context,
+    pass_context,
+    PPLResultStorage,
+    PPLResultIterator,
+    multi_classification,
+    get_data_loader_by_sharding,
+)
+
+from .model import Net
+
+ROOTDIR = Path(__file__).parent.parent
+
+
+class CustomPipelineHandler:
+    def __init__(self) -> None:
+        self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+        self.model = self._load_model(self.device)
+        self.evaluation = t.Optional[None]
```

Comment:
what's the `t.Optional[None]` meaning? 

`self.evaluation = t.Optional[None]` -> `self.evaluation = None`

Reply:
already removed this unused property